easybuild/easyconfigs/m/mrc/mrc-1.3.13-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,58 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2025/01
+easyblock = 'CMakeMake'
+
+name = 'mrc'
+version = '1.3.13'
+
+homepage = 'https://github.com/mhekkel/mrc'
+description = """A long, long time ago there existed an operating system that did things         
```

Comment:
This description can probably be reduced to "Maartens Resource Compiler" the current text doesn't really describe the package except a little bit at the end